src/argilla/client/client.py

```diff
@@ -635,13 +640,21 @@ def _load_records_internal(
         task = dataset.task
 
         task_config = {
-            TaskType.text_classification: (SdkTextClassificationRecord, DatasetForTextClassification),
-            TaskType.token_classification: (SdkTokenClassificationRecord, DatasetForTokenClassification),
-            TaskType.text2text: (SdkText2TextRecord, DatasetForText2Text),
+            TaskType.text_classification: (
+                SdkTextClassificationRecord,
+                DatasetForTextClassification,
+                {"id", "inputs", "multi_label"},
```

Comment:
I think the `event_timestamp` field should be returned always. Otherwise, a new value will be created when a record is initialized from a projection, right @davidberenstein1957?